.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,45 +1,29 @@
-<!--- Provide a general summary of your changes in the Title above -->
+<!--- Title: Provide a general summary of your changes in the Title above -->
+<!--- Reviewers: Include @karstens, @GarthDB, @mrcjhicks, @lynnhao -->
 
 ## Description
 
-<!--- Describe your changes in detail -->
+<!--- Describe your changes in detail, including a list of changes -->
```

Comment:
@GarthDB Would you reference this instead of the Jira ticket?

Reply:
I recommend having them in both locations.